kernel/Kconfig

```diff
@@ -841,7 +841,18 @@ config HEAP_MEM_POOL_SIZE
 	  This option specifies the size of the heap memory pool used when
 	  dynamically allocating memory using k_malloc(). The maximum size of
 	  the memory pool is only limited to available memory. A size of zero
-	  means that no heap memory pool is defined.
+	  means that no heap memory pool is defined. Note that if subsystems
+	  specify HEAP_MEM_POOL_MIN_SIZE_* values that are larger than the
+	  value given here, then the heap size will be rounded up to that value
```

Comment:
> A size of zero means that no heap memory pool is defined. Note that if subsystems
> specify HEAP_MEM_POOL_MIN_SIZE_* values that are larger than the
> value given here, then the heap size will be rounded up to that value.

This must be made clearer.
From the current description it's  not fully clear if having `HEAP_MEM_POOL_SIZE=0` and `HEAP_MEM_POOL_MIN_SIZE_<name>=<value>` means that no heap mem pool will be defined or if it means that heap size will be rounded up to `<value>`.

Reply:
Thanks. I've now updated the Kconfig help text.